### ISSUE TITLE
Add an option to unbatch API calls in the downloaded HAR file

### ISSUE THREAD

**Issue by ccastrotrejo** [ehamai/harsanitizer#3]
I don't know if this is possible but just wanted to see if there could be a function/checkbox to unbatch the api requests. This could help to debug easier

**Comment by ehamai** [comment]
@ccastrotrejo Sorry what do you mean by "unbatch" the API requests?  Like in the HAR file?  Or in the inspector?  The inspector is already unbatching them which is why I mentioned it.

**Comment by ccastrotrejo** [comment]
Yes, I was talking about the HAR file. Just knowing that the inspector does it, it also works fine for me.

**Comment by ehamai** [comment]
Yeah I like the HAR file idea though and I think that should be doable.  Thanks @ccastrotrejo!
